frontend/src/metabase/ui/components/icons/Icon/Icon.tsx

```diff
@@ -39,7 +39,17 @@ export const Icon = forwardRef<SVGSVGElement, IconProps>(function Icon(
     />
   );
 
-  return tooltip ? <Tooltip label={tooltip}>{icon}</Tooltip> : icon;
+  return tooltip ? (
+    <Tooltip
+      style={{ whiteSpace: "unset" }}
```

Comment:
We can do better than [this](https://github.com/user-attachments/assets/f9758964-9059-4f50-84eb-89dc835bf233).
@lorem--ipsum did you consider using `max-width` as suggested in the issue report?

Reply:
I have considered it, yes, but I was unsure what value to pick for that, as it is dependant on the context, which is why I favored this approach.

Reply:
Designers are very helpful with that, and often they'll have better ideas on what the expected result could be :)